### ISSUE TITLE
Failures on [io] Make it possible to change the line ending output by `stdout` and `stderr`

### ISSUE THREAD

**Issue by brianquinlan** [dart-lang/sdk#54949]
There are new test failures on [[io] Make it possible to change the line ending output by `stdout` and `stderr`....[pkg:js] Update README and prep for 0.7.1 publish](https://github.com/dart-lang/sdk/compare/fe67789e43641785ecb957190c81be1f224ec929~...770f44d4e9ffe2abf4c27bc0969c6708c54a9b65).

The tests
```
standalone/io/stdout_stderr_test Timeout (expected Pass)
```
are failing on configurations
```
vm-linux-release-simarm
```

**Comment by dcharkes** [comment]
It's timing out on 7 configurations, and flaky on 7 other configs: https://dart-current-results.web.app/#/filter=standalone/io/stdout_stderr_test&showAll

**Comment by dcharkes** [comment]
@brianquinlan Did you take a look at why this test started failing after your change? Is it hanging?

**Comment by brianquinlan** [comment]
No, I don't understand this issue. I'll investigate it this week.

**Comment by dcharkes** [comment]
Approved another flaky -> timeout.

**Comment by bkonyi** [comment]
Approved another flaky -> Timeout.

**Comment by aam** [comment]
https://dart-ci.appspot.com/log/vm-mac-debug-x64/vm-mac-debug-x64/3983/standalone/io/stdout_stderr_test another flake

**Comment by dcharkes** [comment]
Also:

* https://dart-ci.appspot.com/log/vm-eager-optimization-linux-release-ia32/vm-eager-optimization-linux-release-ia32/497/standalone/io/stdout_stderr_test
* https://dart-ci.appspot.com/log/vm-tsan-linux-release-x64/vm-tsan-linux-release-x64/504/standalone/io/stdout_stderr_test

**Comment by alexmarkov** [comment]
flaky -> Timeout

https://dart-ci.appspot.com/log/vm-mac-debug-x64/vm-mac-debug-x64/5015/standalone/io/stdout_stderr_test

**Comment by derekxu16** [comment]
happened again: https://ci.chromium.org/ui/p/dart/builders/ci.sandbox/vm-tsan-linux-release-x64/592/overview

**Comment by derekxu16** [comment]
happened again: https://ci.chromium.org/ui/p/dart/builders/ci.sandbox/vm-eager-optimization-linux-release-ia32/603/overview

**Comment by dcharkes** [comment]
This test is meta-flaking on multiple configurations: https://dart-ci.web.app/#showLatestFailures=false&test=standalone/io/stdout_stderr_test

**Comment by sstrickl** [comment]
Flaky->timeout on https://ci.chromium.org/ui/p/dart/builders/ci.sandbox/vm-win-release-ia32/634/overview

**Comment by aam** [comment]
https://ci.chromium.org/ui/p/dart/builders/ci.sandbox/vm-eager-optimization-linux-release-ia32/713/overview

**Comment by sstrickl** [comment]
flaky -> Timeout on [vm-eager-optimization-linux-release-ia32](https://dart-ci.appspot.com/log/vm-eager-optimization-linux-release-ia32/vm-eager-optimization-linux-release-ia32/750/standalone/io/stdout_stderr_test)
